pkg/testfixtures/storage/postgres.go

```diff
@@ -46,88 +44,36 @@ func (p *postgresTestContainer) GetDatabaseSchemaVersion() int64 {
 // bootstrapped implementation of the DatastoreTestContainer interface wired up for the
 // Postgres datastore engine.
 func (p *postgresTestContainer) RunPostgresTestContainer(t testing.TB) DatastoreTestContainer {
-	dockerClient, err := client.NewClientWithOpts(
-		client.FromEnv,
-		client.WithAPIVersionNegotiation(),
+	ctx := context.Background()
+
+	postgresContainer, err := testcontainerspostgres.RunContainer(ctx,
+		testcontainers.WithImage(postgresImage),
+		testcontainers.WithEnv(map[string]string{
+			"POSTGRES_DB":       "defaultdb",
+			"POSTGRES_PASSWORD": "secret",
+		}),
```

Comment:
I think this is not needed, because of https://github.com/openfga/openfga/pull/1425/files#diff-cfcf8e9f5facba90b22ea3c79e895216a697868d8f3ca9928b617a068a1cf7a4R63-R65

Reply:
nice, removed